cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -1195,6 +1200,14 @@
           ]
         }
       ]
+    },
+    "EnableEFA": {
+      "Fn::Equals": [
+        {
+          "Ref": "EFA"
+        },
+        "compute"
```

Comment:
shall we do a check for not NONE as done for the others so that we don't need to change this if we support additional options?

Reply:
done